fe/src/pages/index.vue

```diff
@@ -1,6 +1,41 @@
 <template>
   <div>
-    WELCOME TO VUE!
-    <BaseDropdown />
+    <nav class="h-[64px] border-b grid place-content-center">Navbar</nav>
+
+    <div
+      class="h-[558px] grid grid-cols-3 grid-rows-1 gap-4 text-center bg-gradient-container"
+    >
+      <div class="flex flex-col justify-center items-center">
+        <IconsBlocksLeft />
+      </div>
+      <div class="flex flex-col justify-center items-center gap-[32px]">
```

Comment:
Make use of tailwind's default class values.
```suggestion
      <div class="flex flex-col justify-center items-center gap-8">
```